docs/Technical-Documentation/diagrams/tdp-environments.drawio

```diff
@@ -0,0 +1 @@
vB/kONkne9uhsVSfUmublTuAx6UVjB0v5lnfGYKKGRKS1W9WmT0j9q9nGC4q3WpOeRSU8SvdZpZGdYI4Xt7+d7D0xRoW3j+4m/XX+FgZYXWNYnxeaZYp9OysG3PnP5V7d/9wl+DE+wZlMs4EpZ/0bTMmMJeiCgfo0AYJql1Tknpu4H0Pg2sgEblXn7kbWfOLoDNQyXLJVp5bhGuNqJ1BsMaIXxCLG/ATbrsnWXXuz6HUMfJIjE0PXxQOt/2Cuqh5h8tXyCDOw+mEGkGkIiEAPHplgV2K6mFLsH9la0tl49NBryeB3oVM45ALwNgbBsSBk2gMLzhmkZUUHoGnaoMVQ46t0BpKa/dDc/Vb1/QasvF8NUDnDNHnqtbVXc0XTfglFY7fKyla03Ut0tXkK4of7LfxjYj/5unk/Ie85ho6DtVsHpcRuhLrURYOlUzCuuaqmXQJDt00r/38jQiaNM/H7/8qcaSx2nyVMg9t9gw9mU4Qp3u/rGF5wLGFCgohEv4Sp/8bRypy2njqYsqbGYntSysMIsRWp9rP/tSL1SqLeitjO5cGEGGxJnWorwpdSlq3IWMCQtSKDgyFJyR6zIk9YCmMQHGtFrMG3BGrVzPaczIipxtNu7LGojv1ZkostxTZK/9dGddRA5TbKsof0dMgyO2KjxgK1rI0aHGrJfrQx7ZJEbPy8+fEOECkTFJ9+4OubSlXdTG3Tof67SbXTYoHQDB6rdsrf5dereELs/lmHPy1JXcWTKQ6mlCtKjrVTOqMiB4eptkQChOq1yMHk657pqpIPIHSgsextaKmjMyuCNvPPW78B2xZSq6FKRMjnPloickS5B+AuLPqaQ/vXZMPP2eio3VRHzVHpqFXtjY9WzqpdbvaEDZMln4p9HuJindu81Ou/9OZ53TBanA0qKo/GA9HsrpDBIRpoTdymTRx5u3kSROuD53Sa2zSfO84xt+nwLr+e5wTYXXaimjSFm0VQ9TF5vW+QPb1tm5JztTTpBl9ssUARlisSS/2arOKbVjjRZTQ4Y5fOPJbXLbt0uiiIpWxnK0Cdmn2dxzI85VAH0f6uY6cm6gqPTdQa73onyqQcMdhgrKq8EkAW3bcVOT3UahYSPxpd+iBohJqgkTCkrg6NYLVfqbJdD72kWx0yP0QbLqA1DW8pAIaODkBNgKtifQ1RCqHbhUmuCMX27ENVF9WN6YiQDirem0Vjjwy3WLfnyM7k1haNZYCEbB8/HY59QdUZlQILCsClbgqf0Bu/J7N4JFYystVugPGt9uJr0BJM9nnfRNAn7IxoCHRoaZNObYOluQDsu/2xJC7B+2JUV3vNoBmOg+p6EQ7XRFBN7KN1twvWKtSBtcb2A9P78Fcg7698+vseD3yJnvchVU6dxxHD9Nzt4/4i/QARvjNIUrmZkuKbTCsMKOiGVBSVloGFbUoiEVh43ssHvv+Ml+46eHeFBvGXmDj2GB9BWBqILIK6iTM6HUci+1jU9MAypq0sADVQrrKt7Aln+TUwAWoStq3PH2FQxIDVJoC7v1WDJEhKL9n25j01SALmqBTdBvQRbpjDOKdQFlV9UK+vp1PH6ebiGk0T4IDs8Bn6fHODnPPSRMWlVfsP1E13/2B8SaylEuIunFeJ+vR0++Dc3P24gu9f10/2g+tC9JDvnxgsBijYTvshWtcCwvlH07sWmCPL8ElE4EOG70gY//TMXD0gyz+35wjismdEgYa0oD7Xr+hkUP4T1/TWAJDhTXWeNtt3z6l7PtsbtxfP3OSbLH31sYQv8qM+V27A+3gj6vFfbHCR0ONfQxqdzM2vxnugEhBscr652QdW5BxdfxzOlrHrBXhyGCdehn0CjnDlDBfcNj/uQQmFg1Byano6lbqwTvV2zOipNhqwvQh1U5H5MRCdU9Crkmby0J6Pw9InO6eZrlHDPSnEk3FuFETkVNSzwXs0PBKdChS9xj24urp+Gsy1gDSkhpp3LmCDvRxDhXjf+kOec4ek4VUaQidp3zfqQR1tHnOIKDERfYBZdcvJc6ZVMsrrm2OJPq6dRwDxWcRGh8zRO2rO4IQ5osFy6Iim0UdQSjpijsn4F81Ne1jhgj4dmXs6DeJ56E+7R1CPrq82ex4gV4FWpN0H8zkRL7oPU1AFkMGdkyza2qWqqEqYbRg+r9gbGW51iNCB7lvt+D6fl+pgDMmhIbXGUL4tPJYIZyIFbG2HvJpVobryBPqzP9sthXt6B7GYVYtMBh02LJ323icd3fci/GjRwNSbtbT4Xcefs6Pn/FhAg8Iw2Gz9epRvEdTl85H4R2mwl21B3uPWb6GCiq2uN6s8BSrdd7VNorjBZIxJ9myw02zYvkdZt7LiaG5pfd2oOkBGwp6L31SYbUdz25+lsj9UDhqSKgfZLGJxCni5dNASCB3KyL4LVV58Qg1W+boWVWeq9k6Dfmw9qz2/vfyDqb3Dlp0LeuUXjiql9mwhwSlSF9ZrDdKGqWkBq6BOxGpWJ9K2Rktcy3bRjGxnp1tJ59Zsk1az4X6wtsFv7n16qRPk1jrWhJX7PMEkW1ZHcgG/5tSgwjtnCl163FwmNFDDd2E5JYs05tByi31Og4WWhaog7sOCuUqDhgdnSldEG2TNhsEyhZSFryT4YJQFmEyZrOh8HyziYm6ojT5skF0abQHVaNiEt4wpWCmRtDSNEGaN8fpOtNtDnEDAu3Yj4xoW5BIBSLDMVNEN8QrrdBqU0hqbvvbQnVw0I2UhG03JCTgSI1a9vcik6zCsPo5N5Ou0R7aUAXfGyuBtPHW+DkEdSVnYc1+8eWdmm8Z+5/GBpEA1G5GFYRURMTTYfmgShc5npkYidFnMEJmc0TL1oiVLWfSGQPQ6qWGV3/F6kO73p7Yn4w1lI4tReWZZPfcDQjs1VKWUNcihyzIl33hzsPyDKbtJfpAz0sewVoym9MoYVz//Ynf5xzUs3KZEcuB3A6PS3p7jyzhKO44faCoW0ePXyEsLRX77Pw==</diagram></mxfile>
```

Comment:
- who are "tanf users"?  is this diagram also trying to capture the data flow ? if so, understand why "tanf users" included, but then i also wonder if we need to label this path. 
![1051p1](https://user-images.githubusercontent.com/63075587/125502642-c3dc4f32-39b6-489b-ad62-56f6e2a26eb9.PNG)

- wondering if the boundary [diagram](https://github.com/raft-tech/TANF-app/tree/raft-tdp-main/docs/Security-Compliance) needs to be updated in a follow-on ticket since the flows aren't consistent (we actually do not describe clamAV at all in the bd md).
![1051p2](https://user-images.githubusercontent.com/63075587/125502787-5102fe67-6af6-4427-9934-7d0c15b3959b.PNG)

- related to the above, because there are many arrows here, it may not be as obvious that developers do not have access to prod. Could point this out in `line 48` of the md. Also wondering if the path highlighted below needs to be labeled as debugging/testing or something. 
![1051p3](https://user-images.githubusercontent.com/63075587/125502977-d60c94bd-bc60-47e5-b77f-da99be202cff.PNG)

Reply:
> who are "tanf users"? is this diagram also trying to capture the data flow ? if so, understand why "tanf users" included, but then i also wonder if we need to label this path.

This diagram really captures flow of *requests*, not really flow of data. This represents the TANF users/data analysts and admins. The connection to GitHub exists because "deploying to prod" is done by admins who merge PRs into the HHS repo.

> wondering if the boundary diagram needs to be updated in a follow-on ticket since the flows aren't consistent (we actually do not describe clamAV at all in the bd md).

Yes I think so.

> related to the above, because there are many arrows here, it may not be as obvious that developers do not have access to prod. Could point this out in line 48 of the md. Also wondering if the path highlighted below needs to be labeled as debugging/testing or something.

Yes I can add that clarification.

Reply:
![tdp-environments](https://user-images.githubusercontent.com/12901172/125987572-6716254b-8883-4af6-bea5-bcb30e5bbecd.png)

Updated with more clarifying labels

Reply:
> > wondering if the boundary diagram needs to be updated in a follow-on ticket since the flows aren't consistent (we actually do not describe clamAV at all in the bd md).
> 
> Yes I think so.

i created a #1104 to track this alongside some other bd-related updates.